contracts/modules/PermissionManager/GeneralPermissionManager.sol

```diff
@@ -101,4 +99,115 @@ contract GeneralPermissionManager is IPermissionManager, Module {
         return allPermissions;
     }
 
+    /**
+    * @notice use to get all delegates
+    * @return address[]
+    */
+    function getAllDelegates() public view returns(address[]) {
+        return allDelegates;
+    }
+
+    /**
+    * @notice use to check if an address is a delegate or not
+    * @param _potentialDelegate the address of potential delegate
+    * @return bool
+    */
+    function isDelegate(address _potentialDelegate) public view returns(bool) {
```

Comment:
This function is public so I prefer the name `checkDelegate()` instead of `isDelegate()`